keyboards/massdrop/ctrl/keymaps/r-pufky/keymap.c

```diff
@@ -0,0 +1,124 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a license header here, please?

Here's a sample:

https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
I originally didn't add the copyright header for `keymap.c` as I lifted that from another config and modified, as it didn't have a copyright I left as is as it wasn't my original work.